cmd/up/up.go

```diff
@@ -258,11 +259,15 @@ func (up *upContext) activate(autoDeploy, build bool) {
 	isRetry := false
 
 	for {
+		// create a new context on every iteration
+		ctx, cancel := context.WithCancel(context.Background())
+		defer up.shutdown(cancel)
```

Comment:
this is going to be called twice if there is a retry (one before retrying, and another one when exiting the process). could it be an issue?

Reply:
I don't think it gets called twice. In my testing (I did okteto up, and then killed the pod), it only got called once.

Reply:
I added a flag to ensure shutdown only runs once, just in case.

Reply:
If I remember well, `shutdown` needs to be called at the end of every loop, to clean the previous loop resources.